x/wasm/ibc.go

```diff
@@ -52,12 +52,13 @@ func (i IBCHandler) OnChanOpenInit(
 				Endpoint:             wasmvmtypes.IBCEndpoint{PortID: portID, ChannelID: channelID},
 				CounterpartyEndpoint: wasmvmtypes.IBCEndpoint{PortID: counterParty.PortId, ChannelID: counterParty.ChannelId},
 				Order:                order.String(),
-				Version:              version,
-				ConnectionID:         connectionHops[0], // At the moment this list must be of length 1. In the future multi-hop channels may be supported.
+				// DESIGN V3: this may be "" ??
```

Comment:
What does this mean? Do we need to fix as `todo`?

Reply:
I found the [commit](https://github.com/CosmWasm/wasmd/pull/824/commits/b4c36e4d7a8ca6ad8d43fe040e1ac8f6e069e0a2) that contain this fix, but I didn't know this mean. However, this comment remains in the latest version.